.github/workflows/performance.yml

```diff
@@ -74,6 +74,34 @@ jobs:
     - name: Test objc quickstart
       run: ([ -z $plist_secret ] || scripts/third_party/travis/retry.sh scripts/test_quickstart.sh Performance)
 
+  spm:
+    # Don't run on private repo unless it is a PR.
+    if: (github.repository == 'Firebase/firebase-ios-sdk' && github.event_name == 'schedule') || github.event_name == 'pull_request'
+    runs-on: macOS-latest
+    strategy:
+      matrix:
+        target: [iOS, watchOS]
+    steps:
+    - uses: actions/checkout@v2
+    - name: Initialize xcodebuild
+      run: scripts/setup_spm_tests.sh
+    - name: Unit Tests
+      run: scripts/third_party/travis/retry.sh ./scripts/build.sh PerformanceUnit ${{ matrix.target }} spm
+
+  spm-cron:
+    # Don't run on private repo.
+    if: github.event_name == 'schedule' && github.repository == 'Firebase/firebase-ios-sdk'
+    runs-on: macOS-latest
+    strategy:
+      matrix:
+        target: [tvOS, macOS, catalyst]
```

Comment:
macOS should likely be deleted here.  Note, this job only gets run nightly so it hasn't been noticed yet.  

Instead it should do other pod lib lint tests like https://github.com/firebase/firebase-ios-sdk/blob/master/.github/workflows/functions.yml#L114

You can test by temporarily commenting out the schedule line above.